R/dp_tags.R

```diff
@@ -373,6 +373,87 @@ get_property_tags <- function(property, proj_years = NULL) {
         func = function() get_default_art_mortality_rates(proj_years)
       )
     ),
+    art_15plus_num_percent = list(
+      "HAARTBySexPerNum MV" = list(
+        func = get_array_data,
+        rows = 3:4,
+        dimensions = get_sex_dimnames,
+        type = "numeric"
+      )
+    ),
+    art_15plus_num = list(
+      "HAARTBySex MV" = list(
+        func = get_array_data,
+        rows = 3:4,
+        dimensions = get_sex_dimnames,
+        type = "numeric"
+      )
+    ),
+    art_15plus_need_art = list(
+      "NeedARTDec31 MV" = list(
+        func = get_array_data,
+        rows = 2:3,
+        dimensions = get_sex_dimnames,
+        type = "numeric"
+      )
+    ),
+    art_15plus_eligibility_threshold = list(
+      "CD4ThreshHoldAdults MV" = list(
+        func = get_tag_data
+      )
+    ),
+    art_eligibility_pop = list(
+      "PopsEligTreat MV" = list(
+        func = get_eligibility_pop_data,
+        rows = 2:8,
+        cols = 2:6
+      )
+    ),
+    median_cd4_init = list(
+      "MedCD4CountInit MV" = list(
+        func = get_tag_data
+      )
+    ),
+    art_dropout = list(
+      "PercLostFollowup MV" = list(
+        func = get_tag_data
+      )
+    ),
+    art_allocation_method = list(
+      "NewARTPatAllocationMethod MV2" = list(
+        func = get_tag_data
+      ),
+      "fallback" = list(
```

Comment:
should these fallbacks be functions or just data? given they take no arguments, they must be replaceable with data directly?

Reply:
The function isn't needed, though sometimes they do depend on model params. I've removed the functions so now these are either just data or single line of code to create data from model params.